clang-tidy/action.yaml

```diff
@@ -25,6 +25,10 @@ inputs:
     description: ""
     required: false
     default: ${{ github.token }}
+  include-eol-distros:
+    description: ""
+    required: false
+    default: "false"
```

Comment:
By the way, this default value is not mandatory because we check if it's `"true"` or not.

```suggestion
```

Reply:
I agree with it. I'll delete